tasks/43_brackets-indexes/brackets-indexes.js

```diff
@@ -0,0 +1,19 @@
+export function findCloseIndex(sentence, beginIndex) {
+  let depth = 0;
+  let endingIndex = beginIndex;
+  for (let [index, element] of sentence
+    .slice(beginIndex)
+    .split("")
+    .entries()) {
+    if (element === "(") {
```

Comment:
It's hard to read which array we're looping through. Pls move it to variable.

Reply:
Done